### PR TITLE
Alien Alloy Sprite Fix 

### PR DESCRIPTION
**Pull request by Englishformula**
## What Does This PR Do
Fixes https://github.com/ParadiseSS13/Paradise/issues/20994
Flags dynamic_icon_state of alien alloy to FALSE, fixing a bug preventing the in-hand and tile sprites from being visible and being used. 

## Why It's Good For The Game
Fixes a bug; Thanks Qwerty for pointing out the issue. 

## Testing
Printed Alien alloy from Protolathe and stacked/unstacked, held in hands. 

## Changelog
:cl:
fix: Alien Alloy is once more visible in-hand and on tiles.
/:cl:

### CODE REVIEW

**Comment by Landerlow** [comment]
How would a late-join (but not manifest hired) janitor obtain a keyring? Or is that just spawned with them because they are not manifest hired? 

Is there a way to make  a replacement key ring? Or a way for only command to print a replacement? (You could do this perhaps to secway/pimpin ride keys, too, IDK but that's another thing entirely)

**Comment by Landerlow** [comment]
Another question: Are they physical or 'electric' keys? (If they are 'electric', what would emagging them do, if anything? Perhaps all-access when e-magged?)

**Comment by Landerlow** [comment]
Bruh. I commented on the wrong PR. I have no idea how that happened. Please ignore.

**Comment by Henri215** [comment]
Include "Fixes https://github.com/ParadiseSS13/Paradise/issues/20994" into your PR description, it will automatically close the issue when merged.

**Review by Sirryan2002 — APPROVED** [review]

**Comment by SteelSlayer** [comment]
Looks good, just needs deconflicting.

File: code/game/objects/items/stacks/sheets/mineral.dm [hunk 1]
```diff
@@ -425,7 +425,7 @@ GLOBAL_LIST_INIT(plastitanium_recipes, list(
 	name = "alien alloy"
 	icon = 'icons/obj/abductor.dmi'
 	icon_state = "sheet-abductor"
-	item_state = "sheet-abductor"
```
Comment (SteelSlayer) [inline code/game/objects/items/stacks/sheets/mineral.dm:428]: This probably got removed in the deconflicting. You need to re-add this line.

Comment (Englishformula) [reply code/game/objects/items/stacks/sheets/mineral.dm:428]: Apologies, it's fixed now. Thanks!  @SteelSlayer

**Review by S34NW — APPROVED** [review]

**Pull request merged**